src/scripts/devhub.zig

```diff
@@ -129,6 +129,35 @@ fn devhub_metrics(shell: *Shell, cli_args: CLIArgs) !void {
         "checksum message size max",
         "us",
     );
+    const format_time_ms = blk: {
+        timer.reset();
+
+        try shell.exec(
+            "./tigerbeetle format --cluster=0 --replica=0 --replica-count=1 datafile",
+            .{},
+        );
+
+        break :blk timer.read() / std.time.ns_per_ms;
+    };
+    const startup_time_ms = blk: {
+        timer.reset();
+
+        var process = try shell.spawn(
+            .{
+                .stdin_behavior = .Pipe,
+                .stdout_behavior = .Pipe,
+                .stderr_behavior = .Inherit,
+            },
+            "./tigerbeetle start --addresses=0 --cache-grid=8GiB datafile",
+            .{},
+        );
+        errdefer _ = process.kill() catch unreachable;
```

Comment:
Should be defer? Or we should close `process.stdin` on the happy path.

Reply:
Still there^

Reply:
Whoops!

Reply:
Yeah, this one was the only one I _wanted_ to fix before merging 😅